modules/references/client/api/references.api.js

```diff
@@ -76,6 +76,24 @@ export async function read({ userFrom, userTo }) {
   );
 }
 
+/**
+ * API request: read references written by loggedIn user, filter them by userTo,
+ * and sort by 'created' field starting from the most recent date
+ *
+ * @param {string} userTo - id of user who received the reference
+ * @returns Promise<Reference[]> - array of the found references
+ */
+export async function readMine({ userTo }) {
+  const params = { userTo: userTo };
```

Comment:
This can be shortened to:

```js
const params = { userTo };
```

https://alligator.io/js/object-property-shorthand-es6/

Reply:
Added a lint rule to guide us automatically: https://github.com/Trustroots/trustroots/pull/1927